index.bs

```diff
@@ -651,6 +659,7 @@ authorizing an authenticator with which to complete the operation.
         USVString                            rpId;
         sequence<ScopedCredentialDescriptor> excludeList = [];
         Attachment                           attachment;
+        boolean                              verificationGesture;
```

Comment:
naming suggestion: how about "requireUserVerification" instead of "verificationGesture". A "verificationGesture" doesn't seem to be a thing that's a boolean, but (whether or not we) "require" feels more like a boolean.